Tests/SentryTests/Integrations/TestNotificationCenter.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+class TestNotificationCenter {
```

Comment:
M: should this class also have the build type flags for using the proper notification classes?
eg `UIApplication` or `NSApplication` iirc